journals/apps/journals/blocks.py

```diff
@@ -34,10 +44,17 @@ def value_for_form(self, value):
 
 
 class PDFBlock(blocks.StructBlock):
-    '''PDFBlock component'''
+    """PDFBlock component"""
     title = blocks.CharBlock()
     doc = DocumentChooserBlock()
 
+    @mark_safe
+    def render(self, value, context=None):
+        return BLOCK_FORMATTER.format(
+            span=get_unique_span(doc_type=STREAM_DATA_DOC_FIELD, doc_id=value.get(STREAM_DATA_DOC_FIELD).id),
```

Comment:
why we have to pass `doc_type ` and `doc_id` as keyword arguments?

Reply:
`get_unique_span` just required these two arguments, I just passed them as *keyword* argument to increase readability.